services/basic_types.proto

```diff
@@ -1623,21 +1628,9 @@ message GrantedNftAllowance {
     TokenID token_id = 1;
 
     /**
-     * The account ID of the token allowance spender.
+     * The account ID of the spender that has been granted access to all NFTs of the owner
      */
     AccountID spender = 2;
-
-    /**
-     * The list of serial numbers that the spender is permitted to transfer.
-     */
-    repeated int64 serial_numbers = 3;
-
-    /**
-     * If true, the spender has access to all of the account owner's NFT instances (currently
-     * owned and any in the future). If this field is set to true the serialNumbers field
-     * should be empty.
-     */
-    bool approved_for_all = 4;
```

Comment:
Why can't we keep approvedForAll in the query? It should be associated with the spender account and performant to retrieve.

Reply:
No point in having that @steven-sheehy, it will always be true.